### ISSUE TITLE
Sharing the annotations?

### ISSUE THREAD

**Issue by hbredin** [ironiksk/pyannote-db-librispeech#1]
It took me some time but I am finally looking at your plugin. Thanks a lot for that!

I would be happy to have it under the [pyannote Github organization](github.com/pyannote) umbrella and make it *pip-installable* if you are OK with this.

However, before that, it would make things much easier if the annotations were part of the package (instead of having each of us regenerate them). It would also avoid any differences between different installation of `pyannote.db.librispeech` (hence defeating its original purpose of making research reproducible).
 
According to [Librispeech website](http://www.openslr.org/12/), the data was released under CC BY 4.0 license. The license terms are available [here](https://creativecommons.org/licenses/by/4.0/).
In a nutshell, if we use this license as well for `pyannote.db.librispeech`, I think it is OK to have the MDTM files directly in Librispeech/data directory. What do you think?

**Comment by ironiksk** [comment]
Hello Hervé, 

thanks for the interests in my plugin. I like your idea, so let's make this package public available :)

I will push the annotations of the LibriSpeech dataset asap.

**Comment by hbredin** [comment]
Thanks. 

Looking at the annotations, I realize that the 'wav' file naming convention is different from the one used in the original 'flac' database  (e.g. `LibriSpeech/train-clean-100/103/1240/103-1240-0000.flac` is changed into something like `ConvertedLibriSpeech/103-KarenSavage-1240.wav`)

Is there a particular reason why you did that?
Don't you think it would be less error-prone if each `/blah/blah/blah.flac` file was simply renamed into `/blah/blah/blah.wav` file, keeping the original LibriSpeech file structure:

```
<corpus root>
    |
    .- train-clean-100/
                   |
                   .- 19/
                       |
                       .- 198/
                       |    |
                       |    .- 19-198-0001.flac
                       |    |
                       |    .- 14-208-0002.flac
                       |    |
                       |    ...
                       |
                       .- 227/
                            | ...
```


This would also have the additional advantage that people who wants to work directly on `flac` files could simply do something like this:

```
$ cat ~/.pyannote/db.yml 
LibriSpeech:
  - /path/to/LibriSpeech/{uri}.flac
```

Switching to this would obvisouly mean to change the uri field in the annotation files to use the relative path to LibriSpeech, e.g. `uri=train-clean-100/103/1240/103-1240-0000`).

What do you think?
